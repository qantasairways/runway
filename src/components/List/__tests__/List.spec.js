import React from 'react';
import { mount } from 'enzyme';

import { List } from '..';

jest.mock('../../ListItem', () => () => <li>Test</li>);

describe('List', () => {
  it('should match the snapshot when no children are given', () => {
    const component = mount(<List />);

    expect(component).toMatchSnapshot();
  });
});
