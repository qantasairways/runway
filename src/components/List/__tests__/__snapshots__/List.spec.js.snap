// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should match the snapshot when no children are given 1`] = `
.emotion-0 {
  min-width: 240px;
  border-radius: 4px;
  box-shadow: 0 1px 2px 0 rgba(0,0,0,0.2);
  border: solid 1px #dadada;
  background-color: #ffffff;
  list-style: none;
  position: absolute;
  z-index: 5;
  margin: 0;
  top: 0;
  left: 0;
  width: 100%;
  box-sizing: border-box;
  padding: 10px;
}

<List>
  <ul
    className="emotion-0"
  />
</List>
`;
